libs/core/src/components/pds-select/stories/pds-select.docs.stories.mdx

```diff
@@ -0,0 +1,101 @@
+import { Meta, Story, Canvas, Source, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+import * as stories from './pds-select.stories.js';
+import * as optionStories from '../pds-select-option/stories/pds-select-option.stories.js';
+
+<Meta
+  title="components/Select"
+  component="pds-select"
+  argTypes={extractArgTypes('pds-select')}
+  args={{
+    checked: false,
+    disabled: false,
+    invalid: false,
+    required: false,
+  }}
+  subcomponents={{
+    PdsSelectOption: 'pds-select-option',
+  }}
+/>
+
+# Select
+
+Select allows users a single selection or multiple selections from a list of options.
```

Comment:
_suggestion_
Select allows users to make a single or multiple selections from a list of options.